src/sass/modules/buttons.scss

```diff
@@ -204,3 +204,26 @@
     color: $black-80;
   }
 }
+
+.button--checkbox {
+  .fa-check-square {
+    color: color(green);
+  }
+
+  .fa-square {
+    color: $black-80;
+
+    &:hover {
+      color: $black-20;
+    }
+  }
+
+  &:disabled {
+    cursor: default;
```

Comment:
is cursor even needed?

Reply:
Yeah, cursor is needed because our reset scss sets the cursor for all buttons to pointer.